acceptance/bundle/artifacts/cleanup/script

```diff
@@ -0,0 +1,9 @@
+trace $CLI bundle deploy
+
+title "Expecting delete request to artifact_path/.internal folder"
+trace jq -s '.[] | select(.path=="/api/2.0/workspace/delete") | select(.body.path | test(".*/artifacts/.internal"))' out.requests.txt
+
+title "Expecting mkdirs request to artifact_path/.internal folder"
+trace jq -s '.[] | select(.path=="/api/2.0/workspace/mkdirs") | select(.body.path | test(".*/artifacts/.internal"))' out.requests.txt
```

Comment:
I think we can add this queries to one or more of the existing tests, e.g. whl_explicit which has the same config.

Reply:
Makes sense, moved https://github.com/databricks/cli/pull/2572/commits/7894c4c2a7f710c0da54c0121ce9a6d526085c0f